docs/podman-create.1.md

```diff
@@ -710,10 +710,13 @@ Network Namespace - current sysctls allowed:
 
 Note: if you use the --network=host option these sysctls will not be allowed.
 
-**--systemd**=*true|false*
+**--systemd**=*true|false|always*
 
 Run container in systemd mode. The default is *true*.
 
+The value *always* enforces the systemd mode is enforced without
+looking at the executable name.
+
 If the command you running inside of the container is systemd or init, podman
```

Comment:
You need to fix this line to:
If the command you running inside of the container is systemd or /usr/sbin/init, podman

Reply:
s/Podman/podman/